ui/index.html

```diff
@@ -163,6 +163,9 @@ <h1 class="page-header">Dashboard</h1>
 	<!-- Include jquery the electron way -->
 	<script>
     window.jQuery = window.$ = require('jquery');
+    require( 'datatables.net' )( window, $ );
```

Comment:
Can you please remove the extra space between the parentheses and its content here and below?